src/logs.ts

```diff
@@ -134,6 +134,11 @@ const oomAnalyzer: Analyzer = async (text) => {
       'Out of Memory',
       'Allocating more RAM to your instance could help prevent this crash.',
     ];
+  } else if (text.includes('-805306369')) {
```

Comment:
considering we are checking for out of memory above already, wouldn't it make sense to just or this in the if above?
`text.includes('java.lang.OutOfMemoryError') || text.includes('-805306369')`

Also I think it would be best to include the `Error Code:` or whatever the JVM spits out in order to avoid false flags if this number were to show up elsewhere randomly

Reply:
That sounds great! (I don't have much experience in programming, let alone JS.

Reply:
Not quite sure on how to implement showing the error code.